postgres/v15/Dockerfile

```diff
@@ -83,10 +83,10 @@ RUN set -eux; \
     sed -ri "s!^#?(listen_addresses)\s*=\s*\S+.*!\1 = '*'!" /usr/share/postgresql/postgresql.conf.sample; \
     grep -F "listen_addresses = '*'" /usr/share/postgresql/postgresql.conf.sample
 
-RUN mkdir -p /var/run/postgresql && chown -R postgres:postgres /var/run/postgresql && chmod 2777 /var/run/postgresql
+RUN mkdir -p /var/run/postgresql && chown -R postgres:postgres /var/run/postgresql && chmod 1777 /var/run/postgresql
```

Comment:
Is 777 the right permissions? That means anyone can read, write, or execute

Reply:
This should be fine in this instance, because these permissions get updated by the entrypoint script when the image gets started. When we run the audit script for our images however we bypass the entrypoint script, so these permissions come into play. I also looked into the official postgres docker image and it is set up the same way.